packages/page-pattern-modal/src/pattern-definition.ts

```diff
@@ -12,8 +14,16 @@ export interface PatternDefinition {
 	description?: string;
 	html?: string;
 	modified_date?: string;
-	pattern_meta?: { is_mobile: boolean; is_web: boolean };
+	pattern_meta?: { is_mobile: boolean; is_web: boolean; viewport_width: number };
 	site_id?: number;
 	source_url?: string;
 	tags?: Record< string, { slug: string; title: string; description: string } >;
 }
+
+export interface FormattedPattern {
+	name: string;
+	title: string;
+	description: string;
```

Comment:
Do we ever use this `description` prop anywhere? Maybe we can remove it.

Reply:
Removed the description. We have to keep `title` as it's used by [BlockPattern](https://github.com/WordPress/gutenberg/blob/6888113543212f5ceb29edd2cbfb184fece507e8/packages/block-editor/src/components/block-switcher/pattern-transformations-menu.js#L125)